tests/build_tests/super_errors/expected/jsx_custom_component_children.res.expected

```diff
@@ -0,0 +1,14 @@
+
+  [1;31mWe've found a bug for you![0m
+  [36m/.../fixtures/jsx_custom_component_children.res[0m:[2m24:28-29[0m
+
+  22 [2m│[0m }
+  23 [2m│[0m 
+  [1;31m24[0m [2m│[0m let x = <CustomComponent> {[1;31m1.[0m} </CustomComponent>
+  25 [2m│[0m 
+
+  This has type: [1;31mfloat[0m
+  But children passed to this component are expected to have type:
```

Comment:
I understand why @cknitt corrected "is" to "are", but I still find it a bit confusing to use the plural for something that can be singular so we could maybe use a more neutral verb
```suggestion
  But children passed to this component must be of have type:
```
or we could even keep the default wording:
```suggestion
  But the component prop children is expected to have type:
```
But definitely nitpicking here!